h2o-docs/src/product/data-science/gam.rst

```diff
@@ -121,15 +282,49 @@ Common parameters
 - `stopping_rounds <algo-params/stopping_rounds.html>`__: Stops training when the option selected for ``stopping_metric`` doesn't improve for the specified number of training rounds, based on a simple moving average. To disable this feature, specify ``0`` (default). 
 
     **Note:** If cross-validation is enabled:
+
   
     - All cross-validation models stop training when the validation metric doesn't improve.
     - The main model runs for the mean number of epochs.
     - N+1 models may be off by the number specified for ``stopping_rounds`` from the best model, but the cross-validation metric estimates the performance of the main model for the resulting number of epochs (which may be fewer than the specified number of epochs).
 
+
+Example of specifying these parameters:
+
 - `stopping_tolerance <algo-params/stopping_tolerance.html>`__: Specify the relative tolerance for the metric-based stopping to stop training if the improvement is less than this value. This option defaults to ``0.001``.
 
+
 -  `training_frame <algo-params/training_frame.html>`__: *Required* Specify the dataset used to build the model. 
     
+
+   hyper_parameters = {
+     'alpha':[0.9, 0.1],
+     'subspaces': [{
+         'scale':[[1,1,1],[0.1, 0.1, 0.1]],
+         'num_knots':[[4,4,4],[5,5,5]],
+         'gam_columns':[[“C1”,“C2",“C3”]]
+     }, {
+         'scale':[[1,1],[0.1, 0.1,]],
+         'num_knots':[[4,4],[5,5]],
+         'gam_columns':[[“C2",“C3”]]
+     }]
+   }
+
+
+With this hyperparameter specification, 16 GAM models will be built.
+
+-  **num_knots**: Include an array specifying the number of knots for each predictor in ``gam_columns``.
+            
+-  **knot_ids**: Provide a string array of frame keys/IDs containing knot locations. Include one value for each GAM column in ``gam_columns``.
+
+-  **gam_columns**: Required. Include an array of column names representing the smoothing terms used for prediction. GAM will build a smoother for each specified column.
+
+-  **bs**: Add an array specifying the B-Splines for each GAM predictor. Include one value for each GAM predictor. The default value is `0`, which specifies a cubic regression spline.
```

Comment:
please change

 the B-Splines

to

spline types

Reply:
Hannah changed it. Thanks Hannah!